packages/utils/src/random.ts

```diff
@@ -0,0 +1,16 @@
+export const createGUID = (): string => {
```

Comment:
I think renaming to randomNumberGenerator.ts would be more precise

Reply:
`createGUID` returns a string, so this would be confusing. Maybe, `randomGenerator` or something?